pkg/cmd/run_help.go

```diff
@@ -210,7 +210,7 @@ func applyOption(config *RunConfigOption, integrationSpec *v1.IntegrationSpec,
 			return fmt.Errorf("you cannot provide a file larger than 1 MB (it was %s MB), check configmap option or --volume instead", printSize)
 		}
 		// Don't allow a binary non compressed resource
-		rawData, contentType, err := loadRawContent(config.Name())
+		rawData, contentType, err := loadRawContent(context.Background(), config.Name())
```

Comment:
Would it be possible to use the command context, in case it's cancelled?

Reply:
done, mind having a look ?

Reply:
I'm not sure I see the change. I was suggesting replacing `context.Background()` with the command context.

Reply:
I think this view is somehow outdated: https://github.com/apache/camel-k/pull/2780/files#diff-42b6993d49424c3c3f6975bdd9e28d7ef88012ba1a408620edea12fa82459376R213

Reply:
OK I've found it, thanks 👍🏼.